8.1/startup/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Express-driven static site hosted from default App Service wwwroot directory",
   "main": "default-static-site.js",
   "dependencies": {
+    "applicationinsights": "0.21.0",
```

Comment:
I decided to add the AI SDK dependency here, since the Dockerfile was already running `npm install` in this directory, and it actually seemed logical to couple any "in-box" dependencies with the `startup` directory.